### ISSUE TITLE
Rename config settings in datafed.sh

### ISSUE THREAD

**Issue by dvstans** [ORNL/DataFed#849]
Rename:
DATAFED_ZEROMQ_SESSION_SECRET to DATAFED_WEB_SESSION_SECRET
DATAFED_ZEROMQ_SYSTEM_SECRET to DATAFED_SYSTEM_SECRET

These two settings are not related to zeromq

**Comment by JoshuaSBrown** [comment]
Agreed, this is an improvement, we need to thoroughly define these configuration settings in our documentation too, I keep forgetting how they are being used.
